target/linux/layerscape/patches-4.4/0001-arch-Introduce-post-init-read-only-memory.patch

```diff
@@ -0,0 +1,114 @@
+From 7ece66e296972744ec471d521b8ca07990e2494f Mon Sep 17 00:00:00 2001
```

Comment:
I do not like all these patches being added you are ending up with 835 patches on top of kernel 4.4. Please do the migration to kernel 4.9 then you can probably get rid of many patches.